balancer/grpclb/grpclb_remote_balancer.go

```diff
@@ -362,11 +363,35 @@ func (ccw *remoteBalancerCCWrapper) callRemoteBalancer() (backoff bool, _ error)
 	return false, ccw.readServerList(stream)
 }
 
+func (ccw *remoteBalancerCCWrapper) restartRemoteBalancerCall() {
```

Comment:
Name this `cancelRemoteBalancerCall()`
And add a comment where this is called, that a monitoring goroutine will restart the call.